pump/main/views.py

```diff
@@ -36,11 +36,13 @@ class EmailView(View):
     def post(self, request):
         address = request.POST.get('email', None)
         r = get_object_or_404(Response, pk=request.POST.get('response', ''))
-        mail.send_mail(
+        msg = EmailMultiAlternatives(
             'Your PUMP results',
             r.email_text(),
             settings.SERVER_EMAIL,
             [address],
         )
+        msg.attach_alternative(r.email_html(), "text/html")
+        msg.send()
         return HttpResponse(
```

Comment:
imho, this should direct to a nicely formatted success view. Easy enough to setup with a TemplateView.

Reply:
Or, just a redirect back to the results page w/a Django user message at the top.